Makefile

```diff
@@ -8,12 +8,25 @@ doc:
 	@echo Generated documentation: "file://"$$(readlink -f doc/build/html/index.html)
 	@echo
 
+doc3:
+	python3 setup.py build_sphinx
+	@echo
+	@echo Generated documentation: "file://"$$(readlink -f doc/build/html/index.html)
+	@echo
+
```

Comment:
I don't know what `tox` is.  Do whatever you think is reasonable for your development needs.